Applications/Games/Subnautica/Steam/script.js

```diff
@@ -17,22 +14,21 @@ var installerImplementation = {
             .author("Zemogiter")
             .applicationHomepage("https://unknownworlds.com/subnautica/")
             .wineDistribution("upstream")
-            .wineVersion(3.9)
+            .wineVersion("3.16")
```

Comment:
Do you want 3.16 explicitly or can you use the latest development version as well?

Reply:
There is a chance the next development version might introduce some regressions to this game. I'd rather test it beforehand. If it works then someone will make a PR.

Reply:
Ok.